src/styles/block/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  height: $header-height;
+
+  display: flex;
+  justify-content: space-between;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px #0000000d;
+  white-space: nowrap;
+
+  &__link-logo {
+    display: block;
+    height: $header-logo-height;
+    margin-block: $logo-margins;
```

Comment:
The `margin-block` property is not widely supported in all browsers. To ensure cross-browser compatibility, consider using `margin-top` and `margin-bottom` instead.